spec/apc_spec.rb

```diff
@@ -30,17 +30,8 @@
         expect(chef_run).to include_recipe('build-essential')
       end
       it do
-        expect(chef_run).to create_directory('/etc/php.d')
-      end
-      it do
-        expect(chef_run).to create_template('/etc/php.d/APC.ini').with(
-          source: 'apc.ini.erb',
-          owner: 'root',
-          group: 'root',
-          mode: '0644',
-          variables: {
-            params: chef_run.node['osl-php']['apc'],
-          }
+        expect(chef_run).to add_php_ini('APC').with(
+          options: chef_run.node['osl-php']['apc'].merge('extension' => 'apc.so')
```

Comment:
I'd rather you statically test each attribute instead of pulling it from the node attribute. Just in case this doesn't catch issues later on.

Reply:
done